CONTRIBUTING.md

```diff
@@ -1,3 +1,6 @@
 See the [Contributor Guide](https://github.com/magma/magma/wiki/Contributor-Guide/)
 for information on developer onboarding, workflow, contribution conventions,
 presubmit checks and more.
+
+Please read also the [DCO](./DCO.md) for details regarding the legal
```

Comment:
I'm not sure 'legal requirements' is the best term to use here, would be good to double check.

Reply:
Agreed, but I also had no better term for that.

Reply:
@maxhbr 

Instead of `details regarding the legal requirements for a contribution.` we could say:

`details regarding the sign-off rules for a contribution.`

Rules is not as strong as legal requirements.

Reply:
@rdefosse, I applied your suggestion